README.md

```diff
@@ -38,37 +38,42 @@ To make an authorized request, add the following header, where {token} is the `a
 ### Read:
 
 Send a `GET` request to the endpoint. The response will be in JSON format.
-Examples: `http://localhost:3030` + `/event`
+Examples: `http://localhost:3030` + `/events`
 Retrieve everything inside the `events` collection:
 
 -   **Method:** `GET`
--   **Endpont:** `/event`
+-   **Endpont:** `/events`
 
 Retrieve entry with ID: `654651caf696083cab72ab1c` from the events collection:
 
 -   **Method:** `GET`
--   **Endpont:** `/event/654651caf696083cab72ab1c`
+-   **Endpont:** `/events/654651caf696083cab72ab1c`
 
-Retrieve entries by `category` from the events collection:
+Filterin entries by `category` and `region` from the events collection:
+If you want to use filtering you must provide a query string with `category`, `region` or both properties to filter entries.
 
 -   **Method:** `GET`
--   **Endpont:** `/event/:category`
+-   **Endpont:** `/events?category=Rally&region=Пловдив`
```

Comment:
Region to be a number from the enum, the category can also be from enum, synchronise with FE